src/components/TableModule/types.ts

```diff
@@ -33,13 +33,13 @@ export interface TableHeader extends TableAlignOptions {
   className?: string;
 }
 
-export interface TableCell extends TableAlignOptions {
+export interface TableCell<Item = any> extends TableAlignOptions {
   valuePath?: string;
-  content?(cell: any): any;
+  content?(cell: Item): any;
```

Comment:
Primary motivation: allow consumers to avoid this argument being `any` if they want extra safety.